integration_tests/pages/visitDetails.ts

```diff
@@ -47,26 +47,26 @@ export default class VisitDetailsPage extends Page {
   visitEmail = (): PageElement => cy.get('[data-test="visit-email"]')
 
   // Visitor Details-1
-  visitorName1 = (): PageElement => cy.get('[data-test="test-visitor-name1"]')
+  visitorName1 = (): PageElement => cy.get('[data-test="visitor-name1"]')
```

Comment:
All these updated names are better having removed the unnecessary `test-`. But when merged, worth running e2e test job in case it has selectors that need updating.